.github/workflows/test_dep_update.sh

```diff
@@ -7,9 +7,17 @@ git branch depupdate
 git checkout depupdate
 git reset --hard origin/master
 json="$(gh pr list --search "author:app/github-actions" --json headRefName,number)"
+if [[ $json == "[]" ]]; then
+    echo "No Dependency update PRs to merge"
+    exit
+fi
 echo "$json" | jq '.[] | .headRefName | @text' | xargs -L1 -- git pull origin
 if nix build .#checks.x86_64-linux.init-example-el; then
-    echo "$json" | jq ".[] | .number | @text" | xargs -L1 -- gh pr merge --squash --delete-branch
+    prs=$(echo "$json" | jq ".[] | .number")
+    for pr in $prs; do
+        sleep 5
+        gh pr merge --squash --delete-branch "$pr"
+    done
```

Comment:
Whenever there's a bunch of PRs to merge at the same time the GitHub API needs some time to update its internal state before allowing another PR to be merged.